routemaster/models/batch.rb

```diff
@@ -26,10 +26,11 @@ class Batch
       attr_reader :uid, :deadline
 
 
-      def initialize(uid:, deadline: nil, subscriber: nil)
+      def initialize(uid:, deadline: nil, subscriber: nil, length: nil)
         @uid        = uid
         @deadline   = deadline
         @subscriber = subscriber
+        @_length    = length
```

Comment:
I may be missing something as I'm just looking at the diff, but what's the purpose of the ivar assignment? I don't see it being used anywhere else.